templates/common/Makefile

```diff
@@ -104,6 +104,10 @@ site/templates:
 site/include:
 	@(ln -s ../include site/include)
 
+site/ebin:
+	@(mkdir -p _build/default/lib/{{name}}/ebin)
+	@(ln -s ../_build/default/lib/nb/ebin site/ebin)
```

Comment:
Hey @jbcorwin, Let's change this line so that `nb` is `{{name}}`. I'll make some further tweaks to this later, but this line needs a change :). Thanks!

Reply:
Changed and committed.